app/components/user-search.cjsx

```diff
@@ -14,7 +14,7 @@ module.exports = React.createClass
       apiClient.type('users').get display_name: "#{value}*", page_size: 10
         .then (users) =>
           opts = for user in users
-            { value: user.id, label: "@#{ user.login }: #{ user.display_name }" }
+            { value: user.id, label: "@#{ user.login }" }
```

Comment:
Roger that, I'll add the display_name back into the label. Also cool to see search improvement in the pipeline.